homeassistant/components/awair/translations/pt-BR.json

```diff
@@ -24,7 +24,7 @@
                 },
                 "description": "Insira novamente seu token de acesso de desenvolvedor Awair."
             },
-            "user": {
+            "cloud": {
```

Comment:
No need to update `translations` folder. The source of truth is lokalise.

For local testing, run `python3 -m script.translations develop` to generate dev translations.